src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,4 @@
+const initialState = 'Under construction';
+const categoriesReducer = (state = initialState) => state;
+
```

Comment:
- [ ] Just as I suggested above, following the project requirement, you need to have an action creator `checkStatus`. also, the action type needs to by in the reducer. :+1: :smile: